python/mxnet/io.py

```diff
@@ -487,32 +487,26 @@ class NDArrayIter(DataIter):
     >>> labels = np.ones([10, 1])
     >>> dataiter = mx.io.NDArrayIter(data, labels, 3, True, last_batch_handle='discard')
     >>> for batch in dataiter:
-    ...    print batch.data[0].asnumpy()
-    ...    batch.data[0].shape
+    >>>         print batch.data[0].asnumpy()
```

Comment:
why change from ... to >>>? I think the former is the correct version for docstring codes